### ISSUE TITLE
Existing AWS I AM policy fails - permissions are not sufficient

### ISSUE THREAD

**Issue by DarwinJS** [mitchellh/packer#3616]
When I use the exact policy documented here: https://www.packer.io/docs/builders/amazon.html (including the role passing information), I get the below error.

Could someone test and update the policy so that it can accomplish a Packer Build?

==> amazon-ebs: Error waiting for password: Error retrieving auto-generated instance password: UnauthorizedOperation: You are not authorized to perform this operation.
==> amazon-ebs: 	status code: 403, request id:

**Comment by rickard-von-essen** [comment]
Thank you for reporting this.

This is a duplicate of #3546 and already fixed on master in #3558. You can read the updated docs on
[Amazon - Using an IAM Instance Profile](https://github.com/mitchellh/packer/blob/master/website/source/docs/builders/amazon.html.md#using-an-iam-instance-profile). The site will be updated when next release is done.

**Comment by AndrewFarley** [comment]
I realize this issue is closed, but I wanted to +1 because the public docs are still outdated

**Comment by rickard-von-essen** [comment]
@AndrewFarley See my comment just above.

**Comment by AndrewFarley** [comment]
@rickard-von-essen I read the above, it is from 4 months ago and public docs have still not been updated.  I used the public docs and they failed, and I had to go searching out your comment and link above before I got things to work properly.  I just wanted to bump this thread to try to get things bumped to get on the main site to save someone else in the future the trouble of doing extra searching.
